### ISSUE TITLE
Invalid OpenVAS version in remote server

### ISSUE THREAD

**Issue by dauchua** [golismero/openvas_lib#4]
Hello golismero team,
I'm creating a network security tool by using your lib, but when i'm connect to the server i got the error : 
"Invalid OpenVAS version in remote server".
I'm installed OpenVAS 7, in ubuntu 14.04 
Thanks you so much :D

**Comment by wcc526** [comment]
recommend the openvas 6 and kali system. It has tested all is ok in the kali system and openvas 6.

**Comment by aphillipo** [comment]
I'm going to take a stab at getting something working with OpenVAS 7 and OMPv5 tonight.

**Comment by cr0hn** [comment]
Hi! Sorry about the delay. Currently the lib doesn't support OpenVas 7. We will add support for it, but I'm sorry to say that I don't know when.

**Comment by aphillipo** [comment]
If you check out my hacked together branch it should give you some details.  I've removed quite a lot of the getters and setters from the underlying report models because I feel they aren't very pythonic and also a lot of the checks these things do that throw away info; I'd rather keep info that we don't undertsand than just message out that we are throwing it away.  My needs are probably different that the design of the library.  I've also made it so that creating a scan and launching one are separate commands, otherwise I can't see how to retrieve the task id in the finish callback.  All very hacky but it works with OV7.
